jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -1274,32 +1275,30 @@ code_blockt java_bytecode_convert_methodt::convert_instructions(
         typecast_exprt::conditional_cast(op[0], method_return_type);
       c=code_returnt(r);
     }
-    else if(statement==patternt("?astore"))
+    else if(bytecode == patternt("?astore"))
     {
       assert(op.size()==3 && results.empty());
       c = convert_astore(statement, op, i_it->source_location);
     }
-    else if(
-      statement == patternt("?store") || statement == patternt("?store_?"))
+    else if(bytecode == patternt("?store") || bytecode == patternt("?store_?"))
     {
       // store value into some local variable
       PRECONDITION(op.size() == 1 && results.empty());
       c = convert_store(
         statement, arg0, op, i_it->address, i_it->source_location);
     }
-    else if(statement==patternt("?aload"))
+    else if(bytecode == patternt("?aload"))
     {
       PRECONDITION(op.size() == 2 && results.size() == 1);
       results[0] = convert_aload(statement, op);
     }
-    else if(statement == patternt("?load") || statement == patternt("?load_?"))
+    else if(bytecode == patternt("?load") || bytecode == patternt("?load_?"))
     {
       // load a value from a local variable
       results[0]=
         variable(arg0, statement[0], i_it->address, CAST_AS_NEEDED);
     }
-    else if(statement=="ldc" || statement=="ldc_w" ||
-            statement=="ldc2" || statement=="ldc2_w")
+    else if(bytecode == BC_ldc || bytecode == BC_ldc_w || bytecode == BC_ldc2_w)
```

Comment:
ldc2?

Reply:
ldc2 doesn't exist, even though that spoils the symmetry.

Reply:
Aha, so it doesn't